main.py

```diff
@@ -130,6 +132,12 @@ def main():
                     config = utils.load_yaml_config(task)
                     task_names.append(config)
 
+        if task_names == []:
```

Comment:
We still want to error out if even a single one of our comma-separated task names is invalid--does this do that, or only error if there are zero valid task names passed?

Reply:
I see. Let me make some adjustments for that. Currently it only raises error if no tasks were detected. So it would still run if there is one task that isn't valid.